label_studio/tests/utils.py

```diff
@@ -390,3 +392,28 @@ def file_exists_in_storage(response, exists=True, file_path=None):
         file_path = export.file.path
 
     assert os.path.isfile(file_path) == exists
+
+
+def mock_feature_flag(flag_name: str, value: bool, parent_module: str = 'core.feature_flags'):
```

Comment:
I find this easier to use than having to create a fixture for each new ff we want to mock the behavior of... lmk if we prefer the fixture-based method for this though, not the hill I'll die one